### PR TITLE
Add installation of keras_applications in the documentation of intall from sources

### LINKED ISSUE #21734: Latest tensorflow build from source requires keras but no where in the docs it is mentioned
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: N/A
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: latest
- **Python version**: 3.5
- **Bazel version (if compiling from source)**: 0.16
- **GCC/Compiler version (if compiling from source)**: gcc (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609
- **CUDA/cuDNN version**: 9.2/7.2
- **GPU model and memory**: NVidia Tesla V100/16 GB
- **Exact command to reproduce**: bazel build --config=opt --config=cuda //tensorflow/tools/pip_package:build_pip_package with default configuration

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Latest tensorflow build from source requires keras but it is not mentioned in the documentation.

screenshot from 2018-08-20 19-06-00
### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

### PR DESCRIPTION
**Pull request by yongtang**
This fix tries to address the issue raised in #21734 where
the installation of keras_applications was not mentioned in the
documentation of "intall from sources".

This fix add the steps of keras_applications install.
This fix fixes #21734.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by martinwicke** [comment]
@fchollet FYI -- I think you mentioned you have a fix for this in mind that wouldn't require keras_applications to be present during build?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @akshaym: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @akshaym: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @akshaym: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yongtang** [comment]
I checked the documentation website and think the issue has been resolved. So this PR is not relevant any more. I will close this PR but do appreciate the help 👍 .

**Comment by yongtang** [comment]
I checked the documentation website and think the issue has been resolved. So this PR is not relevant any more. I will close this PR but do appreciate the help 👍 .
